kie-dmn-core/src/main/java/org/kie/dmn/core/compiler/DMNEvaluatorCompiler.java

```diff
@@ -322,6 +327,63 @@ private DMNExpressionEvaluator compileDecisionTable(DMNCompilerContext ctx, DMNM
         DMNDTExpressionEvaluator dtee = new DMNDTExpressionEvaluator( node, dtf );
```

Comment:
This is already performed on Validation side, so I assume you want me to "repeat" the check also here in case validation is skipped and go-thru compilation directly, correct?